src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -808,6 +808,7 @@ GREENHOUSE_API_KEY = '{{ GREENHOUSE_API_KEY }}'
 {% endif %}
 
 LOCAL_LOGGING_LOGGERS = {{ localsettings.get('LOCAL_LOGGING_LOGGERS', {}) }}
+LOCAL_MIDDLEWARE = {{ localsettings.get('LOCAL_MIDDLEWARE', []) }}
```

Comment:
does this need to update the default value of `LOCAL_MIDDLEWARE` to `[]` instead of `()` for consistency?

Reply:
we could though I don't think it matters much